clang/test/CodeGen/attr-llvmfn.c

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang_cc1 -debug-info-kind=limited -emit-llvm -o - | FileCheck %s
+
+void t1() __attribute__((llvm_fn_attr("custom_attr", "custom_value"), llvm_fn_attr("second_attr", "second_value")));
+
+void t1()
+{
+}
+
+void t2();
+
+void t3() {
+	t2() ____attribute__((llvm_fn_attr("custom_attr", "custom_value"), llvm_fn_attr("second_attr", "second_value")));
+}
+
```

Comment:
1. Check lines
2. existing attributes need to be tested
3. negative tests are missing, e.g., too many/few arguments, values like "foo,bar" or "300qqqq", etc.

Reply:
Yeah sorry, should've marked this draft (still rebuilding to add the correct check lines, but wanted to open PR for visibility and comment from stakeholders re design)

Reply:
@jdoerfert okay tests added and appear working, ready for review